### PR TITLE
#246 review

### PR DESCRIPTION
**Pull request by MemeWizard**
## Developer changelog:
- This is part of review for #246

### CODE REVIEW

File: common/traits/00_traits.txt [hunk 1]
```diff
@@ -6899,113 +6816,15 @@ faith_warrior = {
 				
 				desc = trait_crusader
 			}
-			triggered_desc = {
-				# Warcraft
-				trigger = { faith = { like_islam_religion_trigger = yes } }
-				
-				desc = trait_mujahid
-			}
```
Comment (zumbak04) [inline common/traits/00_traits.txt:6907]: It fits Wastewanders.

File: events/religion_events/great_holy_war_events.txt [hunk 2]
```diff
@@ -3359,41 +3277,8 @@ great_holy_war.0071 = {
...
-			NOR = {
-				# Warcraft
-				like_christianity_religion_trigger = yes
-				like_islam_religion_trigger = yes
-			}
-		}
+	# Warcraft
 
-		custom_tooltip = great_holy_war.0051.generic.start.tt
-	}
```
Comment (zumbak04) [inline events/religion_events/great_holy_war_events.txt:3396]: There's no option now.

File: events/religion_events/great_holy_war_events.txt [hunk 3]
```diff
@@ -3239,41 +3190,8 @@ great_holy_war.0051 = {
...
```
Comment (zumbak04) [inline events/religion_events/great_holy_war_events.txt:3276]: There's no option now.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: events/religion_events/great_holy_war_events.txt (hunk 3 above)
Comment (MemeWizard) [reply events/religion_events/great_holy_war_events.txt:3276]: ＞︿＜???
image

File: events/religion_events/great_holy_war_events.txt (hunk 2 above)
Comment (MemeWizard) [reply events/religion_events/great_holy_war_events.txt:3396]: ＞︿＜???
image

File: common/traits/00_traits.txt (hunk 1 above)
Comment (MemeWizard) [reply common/traits/00_traits.txt]: If you say so

File: events/religion_events/great_holy_war_events.txt [hunk 4]
```diff
@@ -3920,17 +3826,9 @@ great_holy_war.0061 = {
...
-			# Warcraft
-			limit = { like_islam_religion_trigger = yes }
-			
-			house = {
-				add_house_modifier = {
-					modifier = house_beneficiary_won_ghw_jihad
-					years = 30
-				}
-			}
-		}
```
Comment (zumbak04) [inline events/religion_events/great_holy_war_events.txt:3933]: It may fit the Wastewanders.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**